stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/channel/add/AddChannelViewModel.kt

```diff
@@ -112,7 +110,7 @@ class AddChannelViewModel : ViewModel() {
             return
         }
         viewModelScope.launch(Dispatchers.IO) {
-            val currentUserId = chatDomain.user.value?.id ?: error("User must be set before create new channel!")
+            val currentUserId = chatClient.getCurrentUser()?.id ?: error("User must be set before create new channel!")
```

Comment:
Here we are not using `GlobalState`.
I've tested this offline and it appreas safe but stress test this as much as you can.